test/benchmarks/run.sh

```diff
@@ -38,38 +45,49 @@ function cleanup() {
 
 trap cleanup SIGINT SIGTERM
 
-solc="${SOLIDITY_BUILD_DIR}/solc/solc"
-benchmarks_dir="${REPO_ROOT}/test/benchmarks"
-benchmarks=("chains.sol" "OptimizorClub.sol" "verifier.sol")
-time_bin_path=$(type -P time)
+function bytecode_size {
+    bytecode_chars=$(stripCLIDecorations | stripEmptyLines | wc --chars)
```

Comment:
`bytecode_chars` should be local no? Otherwise it is in the global scope.
```suggestion
    local bytecode_chars
    bytecode_chars=$(stripCLIDecorations | stripEmptyLines | wc --chars)
```

Reply:
Yes, good catch.